### ISSUE TITLE
Split of Cifar10 and Cifar100

### ISSUE THREAD

**Issue by taoyang1122** [D-X-Y/AutoDL-Projects#104]
Hi, the paper says that Cifar-10 training set is split into train and val set, and Cifar-100 val set is split into val and test set. Could you please tell me where to find the split (that is which image is split into which set)? Thanks for your help.

**Comment by D-X-Y** [comment]
Thanks for your interests.
Please find the `cifar-split.txt`, `cifar100-test-split.txt`, and `imagenet-16-120-test-split.txt` in https://github.com/D-X-Y/AutoDL-Projects/tree/main/configs/nas-benchmark.

**Comment by D-X-Y** [comment]
Here is the examples of how to use them to create the data loader for the splitter dataset: https://github.com/D-X-Y/AutoDL-Projects/blob/main/exps/NATS-Bench/main-tss.py#L56-L174
